style.css

```diff
@@ -788,48 +788,74 @@ hr {
   }
 
   .post-container img {
-    width: 100%;
+    width: 600px;
     height: auto;
   }
 
+  .post-container header {
+    margin-bottom: 24px;
+  }
+
   .post-body {
-    margin-left: 24px;
+    display: flex;
+    flex-direction: column;
+    margin-left: 7%;
+    width: 48%;
+  }
+
+  .post-story {
+    margin-bottom: 24px;
+  }
+
+  .btn {
+    margin-top: 24px;
+    align-self: flex-start;
   }
 
   .card-1 {
     margin-top: 20px;
     background-image: url(./image/card-1.png);
     background-repeat: no-repeat;
+    background-size: cover;
   }
 
   .card-2 {
     margin-top: 20px;
     background-image: url(./image/card-2.png);
     background-repeat: no-repeat;
+    background-size: contain;
+    background-size: cover;
   }
 
   .card-3 {
     margin-top: 20px;
     background-image: url(./image/card-3.png);
     background-repeat: no-repeat;
+    background-size: contain;
+    background-size: cover;
   }
 
   .card-4 {
     margin-top: 20px;
     background-image: url(./image/bcg-img-2.png);
     background-repeat: no-repeat;
+    background-size: contain;
+    background-size: cover;
```

Comment:
- [ ]  Good job working on the previous review, however, the project card is still not properly fixed, Kindly fix the project cards text content to match the Figma design. The text content should align with the bottom of the card.

| Your design  | Figma template |
| ------------- | ------------- |
| ![Capture](https://user-images.githubusercontent.com/85095702/160011097-eb1469a0-5afa-4bed-885a-cf097028c35b.PNG) | ![figma](https://user-images.githubusercontent.com/85095702/160011575-3bcb6c1e-ca21-40e9-9213-c1b38d2ff6f4.png) |

- [ ]  Also as requested by the previous reviewer, please adjust the display of the "See Project" button for all cards in desktop view to match the Figma design. (You can make it so that the button only appears when the card is hovered over and the button has to be inside of the card)

Reply:
The "see project" button the previous reviewer complained about has been fixed, I really don't understand what you mean by this, can you please explain.
I am running out of options  @phelian23 
is it the size i should fix??

Reply:
I have adjusted it.

Reply:
Resolved

Reply:
![image](https://user-images.githubusercontent.com/70589265/160027101-d84f0d58-306e-4955-90c0-17c3f73227c3.png)

Reply:
this is proof that the project is okay based on the first reviewer's comment.